modules/administration-guide/partials/proc_setting-up-the-microsoft-azure-devops-services-oauth-app.adoc

```diff
@@ -46,6 +46,8 @@ $ echo -n '__<microsoft_azure_devops_services_oauth_app_id>__' | base64
 $ echo -n '__<microsoft_azure_devops_services_oauth_client_secret>__' | base64
 ----
 
+. Make sure to authorize the application when Azure DevOps Services presents the authorization approval page to a user.
```

Comment:
@satyamburhade 
Actually I didn't face anything that requires approval.
Could you show some screenshots for better understanding?